spec/book_spec.rb

```diff
@@ -0,0 +1,50 @@
+require './book'
```

Comment:
### Optional
- [ ] It would be better if we use `require_relative` instead of `require` when we are working with files in our current directory. The `require` can be used when we are working with packages in our gemfiles